src/forms/FormFieldPhoneSelectionDropdown/FormFieldPhoneSelectionDropdown.tsx

```diff
@@ -24,6 +24,17 @@ const FormFieldPhoneSelectionDropdown = (
 		value,
 	} = props;
 
+	const [dialCode, setDialCode] = useState("");
+
+	const onPhoneChange = (phoneValue: string, data: CountryData) => {
+		onChange(phoneValue);

```

Comment:
This approach executes the onChange twice when the user deletes all the text, but it's too quick to be noticeable. If you use breakpoints you'll see it goes to the fieldOnChange action twice, which we could avoid to prevent sending data twice.

I suggest either moving the if statement to the top of the function and then return, or do an else statement.